src/type/matrix/DenseMatrix.js

```diff
@@ -458,7 +458,8 @@ export const createDenseMatrixClass = /* #__PURE__ */ factory(name, dependencies
     const m = copy ? this.clone() : this
 
     m._data = reshape(m._data, size)
-    m._size = size.slice(0)
+    const currentLength = m._size.reduce((length, size) => length * size)
```

Comment:
Having a `length` method for matrices might by useful, too.

Reply:
So with length you mean the total number of items in the array? The name `length` may be a bit confusing, but such a function may be useful. Could be a method, but also a standalone function .

Reply:
Yes, the total number of items in the array/matrix.

Here's overview how other libraries call their functions:

| Library | element count | each dimension |
| --- | --- | --- |
| numpy | [`size`](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.size.html) | [`shape`](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.shape.html) |
| Julia | [`length`](https://docs.julialang.org/en/v1/base/arrays/#Base.length-Tuple{AbstractArray}) | [`size`](https://docs.julialang.org/en/v1/base/arrays/#Base.size) |
| mathjs | ? | `size` |

I'm not sure how to call it then... Do you have a good idea?

PHP calls its array length `count`. But using `numberOfElements` or `elementCount` seems a bit long.

Reply:
Let's discuss in the new issue you opened for this: https://github.com/josdejong/mathjs/issues/2085